bundles/persistence/org.openhab.persistence.jpa/java/org/openhab/persistence/jpa/internal/JpaHistoricItem.java

```diff
@@ -111,8 +111,6 @@ public static HistoricItem fromPersistedItem(JpaPersistentItem pItem, Item item)
             state = OpenClosedType.valueOf(pItem.getValue());
         } else if (item instanceof RollershutterItem) {
             state = PercentType.valueOf(pItem.getValue());
-        } else if (item instanceof ColorItem) {
```

Comment:
Explain or revert this change.

Reply:
Again this is a redundant check and dead code, because the check for `ColorItem` comes after the check for its superclass. Here again, we could instead move the check for `ColorItem` before the check for `DimmerItem` if you prefer that.